presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -721,7 +721,7 @@ public static long levenshteinDistance(@SqlType("varchar(x)") Slice left, @SqlTy
     @SqlType(StandardTypes.BIGINT)
     public static long hammingDistance(@SqlType("varchar(x)") Slice left, @SqlType("varchar(y)") Slice right)
     {
-        int distance = 0;
+        long distance = 0;
```

Comment:
I think it's fine to use `int` as distance.  

Even for SQL INTEGER type, we will return `long` , and only the lower 16 bits are used. 

I think one real question is , whether the return SqlType should be `INTEGER` or `BIGINT`.  Note for function `levenshteinDistance`, the `distance` is also int and the return SqlType is `BIGINT`.